site/app/views/LateDaysTableView.php

```diff
@@ -27,4 +27,18 @@ public function showLateTablePage(LateDays $late_days) {
             'grade_inquiry_enabled' => $this->core->getConfig()->isRegradeEnabled()
         ]);
     }
+
+    public function showLateTabletoInstructor(LateDays $late_days) {
```

Comment:
A PHP type declaration for the return type should be added here

Reply:
Return type and parameter added